lib/matplotlib/backends/backend_ps.py

```diff
@@ -83,8 +84,7 @@ def gs_version(self):
             pass
 
         from matplotlib.compat.subprocess import Popen, PIPE
-        s = Popen(self.gs_exe + " --version",
-                     shell=True, stdout=PIPE)
+        s = Popen([str(self.gs_exe), " --version"], stdout=PIPE)
```

Comment:
I believe the extra space here is breaking this version check thus causing your Windows failures.